package.json

```diff
@@ -1,4 +1,7 @@
 {
+  "engines": {
```

Comment:
This can be removed i think. We only want it in the package.json for each package.